package.json

```diff
@@ -27,6 +27,7 @@
     "dagre": "^0.8.5",
     "deepmerge": "^4.2.2",
     "eslint-plugin-only-error": "^1.0.2",
+    "fhir-validator": "^0.5.1",
```

Comment:
This is a good idea but it looks like the library hasn't been updated in a couple years, and only supports STU3. I guess not much has changed in the PlanDefinition/ActivityDefinition resources between 3 and 4.
The other thing is that it doesn't do anything with profiles, so we could be conformant to pure FHIR but violate the CPG IG.

From a quick search I don't see that there's anything better out there, but it's worth digging a little deeper. If we don't find anything I'd suggest we don't use this library because its results are misleading.

Reply:
Wrote some tests and condensed the PlanDefinition methods. Let me know if there's anything else you think the tests should check for

Reply:
@blangley28 is `fhir-validator` still used or should it be removed?  I didn't see it in the diffs.

Reply:
Good catch - it can be removed